parasolr/tests/test_indexing.py

```diff
@@ -50,7 +50,7 @@ def all():
 class AbstractIndexable(Indexable):
     """indexable subclass that should not (itself) be indexed"""
 
-    class Meta:
+    class _meta:
```

Comment:
oh wow okay, so this is what we'll have to do?

Reply:
it's renamed here because this isn't actually a django model subclass — we were referencing `._meta` elsewhere for verbose name but not for the abstract check. I think django's model metaclass does some magic with `.Meta` which is why we weren't getting the value we expected when we checked if it was abstract.